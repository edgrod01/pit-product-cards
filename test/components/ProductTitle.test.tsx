import * as React from 'react';
import renderer from "react-test-renderer";
import { ProductTitle, ProductCard } from '../../src/components';
import { product1 } from '../data/products';


describe( 'ProductTitle', () => {
    test('mostrar e lcompononte titulo personalizado', () => {
        const wrapper = renderer.create(
            <ProductTitle title='Custom product' className='custom-class' />
        )

        expect( wrapper.toJSON() ).toMatchSnapshot();
        
    })

    test('componente con el nombre del producto ', () =>{

        const wrapper = renderer.create(
            <ProductCard product={ product1 }>
                {
                    () => (
                        <ProductTitle />
                    )
                }
            </ProductCard>
        )

        expect( wrapper.toJSON() ).toMatchSnapshot();
    } )
})

