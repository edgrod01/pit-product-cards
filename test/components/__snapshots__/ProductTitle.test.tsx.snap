// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductTitle componente con el nombre del producto  1`] = `
<div
  className="productCard undefined"
>
  <span
    className="productDescription undefined"
  >
    Coffe Mugs
  </span>
</div>
`;

exports[`ProductTitle mostrar e lcompononte titulo personalizado 1`] = `
<span
  className="productDescription custom-class"
>
  Custom product
</span>
`;
